pyanaconda/ui/gui/spokes/datetime_spoke.py

```diff
@@ -352,7 +355,10 @@ def apply(self):
         if not region or not city:
             return
 
-        self._timezone_module.Timezone = region + "/" + city
```

Comment:
You don't have to change this line, the property uses the highest priority by default.

Reply:
I think using the same function everywhere is preferable.